### ISSUE TITLE
Migrate note taking from Markdown to LaTeX

### ISSUE THREAD

**Issue by neilzxu** [joseph-zhong/Papers#2]
LaTeX is a format that is much superior for writing equations and notes.

**Comment by joseph-zhong** [comment]
Yes but the overhead is insane. I'm currently using Typora to render markdown to math, although in general I believe pandoc does the rendering well, but for quick notes I think this is effective. More extensive notes like a cheatsheet perhaps could be latex yes... I've experimented with it before and not sure yet how to tie it together (https://github.com/joseph-zhong/Machine-Learning)

**Comment by neilzxu** [comment]
what overhead

**Comment by joseph-zhong** [comment]
I find it difficult to type latex during lectures or meetings for example as there is a lot of syntax to type, even for relatively simple things such as lists, lists, or bold font, while really I only care about expressiveness for math equations and graphs, which can be refined offline

**Comment by neilzxu** [comment]
maybe you should get better latex/ use custom macros to simplify things.
